src/components/vmd-appointment-card.component.ts

```diff
@@ -178,13 +181,14 @@ export class VmdAppointmentCardComponent extends LitElement {
             `);
     }
 
-    connectedCallback() {
-        super.connectedCallback();
-        // console.log("connected callback")
+    private cardTitle(cardConfig: any): string {
+      if (this.lieu.prochain_rdv) {
+        return this.toProperCase(formatDate(parseISO(this.lieu.prochain_rdv), "EEEE d MMMM 'à' HH:mm", { locale: fr }))
+      } else {
+        return cardConfig.libelleDateAbsente
+      }
     }
-
-    disconnectedCallback() {
-        super.disconnectedCallback();
-        // console.log("disconnected callback")
+    private toProperCase(date: string): string {
+      return date.replace(/(^|\s)([a-z])(\w)/g, (_, leader, letter, loser) => [leader, letter.toUpperCase(), loser].join(''))
```

Comment:
Est-ce que tu as un exemple de ce que fait cette méthode ?

Reply:
`"kikoo les amis à batons de bois!" -> "Kikoo Les Amis à Batons De Bois!"`
c'est parce que date-fns sort les dates avec des minuscules uniquement et qu'elle sont en majuscule à l'heure actuelle

Reply:
Je m'étais fait la même réflexion que @Luwangel : je ne connaissais pas la sémantique "Proper case" (je connaissais Pascal/Snake/Camel/Kebab mais pas Proper) et du coup je ne pensais pas que c'était le nom d'un casing.

Ça, plus le fait que l'implémentation ne me parlait pas énormément, une implem un peu plus "évidente" ne serait-elle pas mieux ?

J'aurais bien vu un truc comme :
```suggestion
      return date.split(' ').map(s => `${s[0].toUpperCase()}${s.substr(1)}`).join(' ');
```

Qu'en penses-tu ?

Reply:
ok, ça passe pas la spec car je mets en majuscule le "à" ... hum

Reply:
Est-ce que du coup, en terme de sémantique, on ne serait pas plutôt sur du "Title case" (que je découvre aussi) ?
https://www.computerhope.com/jargon/t/title-case.htm

Reply:
tout à fait